pkgs/misc/sound-of-sorting/default.nix

```diff
@@ -1,23 +1,19 @@
-{ lib, stdenv, fetchFromGitHub, SDL2, wxGTK } :
+{ lib, stdenv, fetchFromGitHub, pkg-config, SDL2, wxGTK32, Cocoa }:
 
 stdenv.mkDerivation rec {
```

Comment:
Please remove the now useless `rec`.

Also update the meta.license to `gpl3Plus`.

Reply:
Done.